feature/google/src/latestDependencies/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -198,6 +199,7 @@ class BillingWrapper(
         activity: Activity,
         appUserID: String,
         storeProduct: StoreProduct,
+        purchaseOption: PurchaseOption,
         replaceSkuInfo: ReplaceSkuInfo?,
```

Comment:
the replacement/upgrade/downgrade flow now requires the purchase token rather than just the sku. i was thinking we could do that lookup for them. so have them pass in productId, call queryPurchases to get the active sub with that id and then find the token in there.

Reply:
Let's create a new ticket for this, I think I forgot that one.
https://revenuecats.atlassian.net/browse/CF-997
But let's not deal with this here

Reply:
I agree - I think we can just link the StoreProduct in the PurchaseOption, right?